doc/source/tutorial/signal.rst

```diff
@@ -1463,34 +1462,33 @@ with a negative slope:
     >>> f0, Sz0 = fftshift(f0_u), fftshift(Sz0_u, axes=0)
     ...
     >>> # New STFT:
-    >>> SFT = ShortTimeFFT.from_window(win, fs, nperseg, noverlap,
-    ...                                fft_mode='centered',
+    >>> SFT = ShortTimeFFT.from_window(win, fs, nperseg, noverlap, fft_mode='centered',
     ...                                scale_to='magnitude', phase_shift=None)
     >>> Sz1 = SFT.stft(z)
     ...
     >>> # Plot results:
     >>> fig1, axx = plt.subplots(2, 1, sharex='all', sharey='all',
     ...                          figsize=(6., 5.))  # enlarge figure a bit
     >>> t_lo, t_hi, f_lo, f_hi = SFT.extent(N, center_bins=True)
-    >>> t_str0 = r"Legacy stft() produces $%d\times%d$ points" % Sz0.T.shape
-    >>> t_str1 = r"ShortTimeFFT produces $%d\times%d$ points" % Sz1.T.shape
-    >>> _ = axx[0].set(title=t_str0, xlim=(t_lo, t_hi), ylim=(f_lo, f_hi))
-    >>> _ = axx[1].set(title=t_str1, xlabel="Time $t$ in seconds " +
-    ...                rf"($\Delta t= %g\,$s)" % SFT.delta_t)
+    >>> axx[0].set_title(r"Legacy stft() produces $%d\times%d$ points" % Sz0.T.shape)
+    >>> axx[0].set_xlim(t_lo, t_hi)
+    >>> axx[0].set_ylim(f_lo, f_hi)
+    >>> axx[1].set_title(r"ShortTimeFFT produces $%d\times%d$ points" % Sz1.T.shape)
+    >>> axx[1].set_xlabel(rf"Time $t$ in seconds ($\Delta t= {SFT.delta_t:g}\,$s)")
     ...
-    >>> # Calculate extent of plot with centered bins since imshow
-    ... # does not interpolate by default:
-    ... dt2 = (nperseg-noverlap) / fs  / 2 # equals SFT.delta_t / 2
-    >>> df2 = fs / nperseg / 2 # equals SFT.delta_f / 2
+    >>> # Calculate extent of plot with centered bins since
+    >>> # imshow does not interpolate by default:
+    >>> dt2 = (nperseg-noverlap) / fs / 2  # equals SFT.delta_t / 2
+    >>> df2 = fs / nperseg / 2  # equals SFT.delta_f / 2
     >>> extent0 = (-dt2, t0[-1] + dt2, f0[0] - df2, f0[-1] - df2)
     >>> extent1 = SFT.extent(N, center_bins=True)
     ...
     >>> kw = dict(origin='lower', aspect='auto', cmap='viridis')
     >>> im1a = axx[0].imshow(abs(Sz0), extent=extent0, **kw)
     >>> im1b = axx[1].imshow(abs(Sz1), extent=extent1, **kw)
     >>> fig1.colorbar(im1b, ax=axx, label="Magnitude $|S_z(t, f)|$")
-    >>> _ = fig1.supylabel(rf"Frequency $f$ in Hertz ($\Delta f = %g\,$Hz)" %
-    ...                    SFT.delta_f)
+    >>> _ = fig1.supylabel(r"Frequency $f$ in Hertz ($\Delta f = %g\,$Hz)" %
```

Comment:
Do we need this?
```suggestion
    >>> fig1.supylabel(r"Frequency $f$ in Hertz ($\Delta f = %g\,$Hz)" %
```

Reply:
This is needed because otherwise `python dev.py -n refguide-check` would produce the following error:
```
File "doc/source/tutorial/signal.rst", line 1490, in /home/dietrich/GitHub/scipy/tools/../doc/source/tutorial/signal.rst
Failed example:
    fig1.supylabel(r"Frequency $f$ in Hertz ($\Delta f = %g\,$Hz)" %
                       SFT.delta_f, x=0.08, y=0.5, fontsize='medium')
Expected nothing
Got:
    Text(0.08, 0.5, 'Frequency $f$ in Hertz ($\\Delta f = 4\\,$Hz)')


ERROR: refguide or doctests have errors
```